credentials/development/fetch-development-paa-certs-from-dcl.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+
+#
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Script that was used to fetch CHIP Development Product Attestation Authority (PAA)
+# certificates from DCL.
+# The script expects the path to the dcld tool binary as an input argument.
+#
+# Usage example when the script is run from the CHIP SDK root directory:
+#     python ./credentials/development/fetch-development-paa-certs-from-dcl.py /path/to/dcld
+#
+# The result will be stored in:
+#     credentials/development/paa-root-certs
+#
+
+import os
+import sys
+import subprocess
+import copy
+import re
+from cryptography.hazmat.primitives import serialization
+from cryptography import x509
+
+
+def parse_paa_root_certs(cmdpipe, paa_list):
+    result = {}
+
+    while True:
+        line = cmdpipe.stdout.readline()
+        if not line:
+            break
+        else:
+            if b': ' in line:
+                key, value = line.split(b': ')
+                result[key.strip(b' -')] = value.strip()
+                parse_paa_root_certs.counter += 1
+                if parse_paa_root_certs.counter % 2 == 0:
+                    paa_list.append(copy.deepcopy(result))
```

Comment:
A quick comment block explaining what type of text manipulation is done, with sample input/sample output, would help in maintaining this code